gammapy/maps/wcsnd.py

```diff
@@ -26,7 +26,7 @@
 
 
 class WcsNDMap(WcsMap):
-    """HEALPix map with any number of non-spatial dimensions.
+    """WCS map with any number of non-spatial dimensions.
```

Comment:
Funny that this hasn't been noticed earlier...does it mean that users never look at docstrings? Or maybe they just don't care...:-)